recipe/meta.yaml

```diff
@@ -20,10 +20,10 @@ requirements:
     - python >=3.8
   run:
     - numpy
-    - openmm >=8.0.0
+    - openmm
     - python >=3.8
-    - openmm-torch >=1.0
-    - torchani >=2.2.2
+    - openmm-torch
+    - torchani
```

Comment:
Why did you remove the version requirements?  It still requires them.

Reply:
Re-added them!